rotkehlchen/tests/api/blockchain/test_optimism.py

```diff
@@ -19,13 +19,11 @@
 TEST_ADDY = '0x9531C059098e3d194fF87FebB587aB07B30B1306'
 
 
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
```

Comment:
Did you mock this as I asked with the old ones? So basically did you run record after: `2023-01-30 11:14:00`?

Reply:
yes, it would fail If I didn't record the previous state

Reply:
ah yes ofc. Since the test has no changes inside it at all.